lib/rules/no-empty-component-block.js

```diff
@@ -85,16 +99,29 @@ module.exports = {
             isValueOnlyWhiteSpacesOrLineBreaks(componentBlock) ||
             componentBlock.children.length === 0
           ) {
-            context.report({
-              node: componentBlock,
-              loc: componentBlock.loc,
-              messageId: 'unexpected',
-              data: {
-                blockName: componentBlock.name
-              }
-            })
+            emptyBlocks.push(componentBlock)
           }
         }
+
+        if (emptyBlocks.length === 0) return
+
+        for (const componentBlock of emptyBlocks) {
+          context.report({
+            node: componentBlock,
+            loc: componentBlock.loc,
+            messageId: 'unexpected',
+            data: {
+              blockName: componentBlock.name
+            },
+            fix: autofix
+              ? function* (fixer) {
+                  for (const componentBlock of emptyBlocks) {
+                    yield fixer.remove(componentBlock)
+                  }
+                }
```

Comment:
Could you please change it to auto-fix only reported block?

```suggestion
              ? (fixer) => fixer.remove(componentBlock)
```